### ISSUE TITLE
`--dir` is ignored for `init` commands?

### ISSUE THREAD

**Issue by rgitzel** [cloudreach/sceptre#467]
If I use `init` from a folder where I have write permissions (say `/tmp/s`), the command succeeds:
```
$ sceptre init project foo
...
$ ls foo
config  templates
```

If I then remove write permission on this folder, the command fails as expected:
```
$ chmod -w .
$ sceptre init project bar
...
PermissionError: [Errno 13] Permission denied: '/tmp/s/bar'
```

So then I try to specify a different folder where I _do_ have write permission... and the commands fails unexpectedly:
```
$ sceptre --dir /home/rodney init project bar
...
PermissionError: [Errno 13] Permission denied: '/tmp/s/bar'
```

So it seems Sceptre is ignoring `--dir` for `init` commands?

**Comment by ngfgrant** [comment]
Hey @rgitzel is this on `master` or in `v1`?

**Comment by rgitzel** [comment]
Ummm... v1 I _think_?

I don't have it installed anymore (I'm using the Docker image), so for sure it's an issue with v1.
```
/tmp # cd r
/tmp/r # sceptre init project foo
Please enter a project_code [foo]:
Please enter a region []:
/tmp/r # ls
foo
/tmp/r # sceptre --dir /tmp init project foo
Folder "foo" already exists.
```

**Comment by ngfgrant** [comment]
Hey @rgitzel would you be willing to test this on our version 2 release candidate? It is easy to install into a fresh virtualenv using `pip install sceptre --pre`. Would be good to know if this issue still exists.

**Comment by ngfgrant** [comment]
fixed in v2.0
